felix/dataplane/linux/int_dataplane.go

```diff
@@ -480,8 +480,8 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		if !config.RouteSyncDisabled {
 			log.Debug("RouteSyncDisabled is false.")
 			routeTableVXLAN = routetable.New([]string{"^vxlan.calico$"}, 4, true, config.NetlinkTimeout,
-				config.DeviceRouteSourceAddress, config.DeviceRouteProtocol, true, unix.RT_TABLE_UNSPEC,
-				dp.loopSummarizer, routetable.WithLivenessCB(dp.reportHealth))
+				config.DeviceRouteSourceAddress, config.DeviceRouteProtocol, true, unix.RT_TABLE_MAIN,
```

Comment:
After reviewing the whole change, I guess it was probably just that I hadn't made the right change widely enough - which you now **have** done.

Reply:
Practically nothing.  I think it may have an impact if your kernel is compiled without multiple routing tables but that's certainly not something we support.

When the kernel sees 0 it defaults it to main anyway.

It does have a real impact on the kernel-side filtering logic though.  0 is the wildcard so the kernel returns all routes from all tables.  Then netlink filters it down with special-case logic in userspace.